lib/rules/no-duplicate-imports.js

```diff
@@ -21,18 +21,107 @@ function getValue(node) {
     return "";
 }
 
+/**
+ * Checks if the import's type is the given type import.
+ * @param {ASTNode} node The import to be checked.
+ * @param {type} type the type to be checked.
+ *
+ * @returns {boolean} Whether or not the import's type is type.
+ */
+function checkImportType(node, type) {
+    return (
+        node.specifiers &&
+        node.specifiers[0] &&
+        node.specifiers[0].type === type
+    );
+}
```

Comment:
We shouldn't check only the first specifier. For example, this shouldn't be reported:

```js
/* eslint no-duplicate-imports: error */

import foo, * as bar from 'mod';

import { baz } from 'mod';
```